WycliffeAssociates/8woc2018-jvm
# Takes View Model- JVM

**What is included:**
- Takes tabpane is now populated with the selected resource's content and takes
- "New Take" button opens the preferred audio plugin and creates a new take for the selected resource
- Only one take can be played at a time. If a take is playing and another take is played, the first one will pause.

**What is not included:**
- Editing, deleting, selecting takes
- Properly formatted markdown (what you see is the raw markdown content)
- Navigation (you can navigate between the resources list and takes page by checking out the kj-takes-testapp branch, but this code is not in the PR.)

**How to test:**
1. Check out `kj-takes-testapp` branch (based on this PR's branch), and run `main` in ResourceTakesApp.kt. Also make sure to checkout the corresponding branch in common, `kj-takes-tabpane-newtake` (sister PR [#105](https://github.com/WycliffeAssociates/otter-common/pull/105) in common)
2. A chapter will be loaded by default, after which you can select "View Recordings" to go to the takes page for any resource. 
3. Try creating new takes
4. Play a take, and try playing a take while another take is playing
5. Switch between the "snippet" and "note" tabs
6. Return to the resource list page by clicking the "back" button in the header. (This header as well as the file menu are not in this PR, they are only in the kj-takes-testapp branch.)
